Tests/LibCompress/TestBrotli.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2022, Michiel Visser <opensource@webmichiel.nl>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <LibCompress/Brotli.h>
+#include <LibCore/Stream.h>
+
+static void run_test(StringView const file_name)
+{
+    // This makes sure that the tests will run both on target and in Lagom.
+    // When using Lagom the tests should be executed from the root of the repository.
+#ifdef __serenity__
+    String path = String::formatted("/usr/Tests/LibCompress/brotli-test-files/{}", file_name);
+#else
+    String path = String::formatted("../../Tests/LibCompress/brotli-test-files/{}", file_name);
+#endif
+
+    auto cmp_file = MUST(Core::Stream::File::open(path, Core::Stream::OpenMode::Read));
+    auto cmp_data = MUST(cmp_file->read_all());
+
+    String path_compressed = String::formatted("{}.br", path);
```

Comment:
Why don't you need the weird path handling here? That looks suspicious.

Reply:
This is just appending the `br` extension at the end of the file path.